lib/travis/api/v3/models/log.rb

```diff
@@ -1,25 +0,0 @@
-module Travis::API::V3
-  class Models::Log < LogsModel
```

Comment:
I'm thinking for V3 since Logs come from either our logs API, and/or S3 and no longer directly from the DB we should have a log class here that can be used in the renderer and wraps whatever the returned information from the query is. Then we will no longer need to special case RemoteLogs in V3, and/or have an odd type.